wski18n/resources/en_US.all.json

```diff
@@ -1586,5 +1586,21 @@
   {
     "id": "A response type of 'http' is required when using path parameters.",
     "translation": "A response type of 'http' is required when using path parameters."
+  },
+  {
+    "id": "help message for deployment",
+    "translation": "To begin working with deploy, run the 'wsk project deploy' command:\n\n\t$ wsk project deploy\n\nThis will deploy OpenWhisk assets specified in manifest.yaml or manifest.yml\nIf current directory doesnt have any manifest file, wsk project deploy prints this help message.\n\nThe most common ways of using wsk project deploy to deploy and/or undeploy OpenWhisk assets are:\n\n$ wsk project deploy\n$ wsk project deploy --manifest path/to/manifest.yaml\n$ wsk project deploy --manifest path/to/manifest.yaml --deployment path/to/deployment.yaml\n$ wsk project undeploy\n$ wsk project undeploy --manifest path/to/manifest.yaml\n$ wsk project undeploy --manifest path/to/manifest.yaml --deployment path/to/deployment.yaml\n\nAunthentication:\nwsk project deploy reads credentials from $HOME/.wskprops by default\nOverwrite default config file on CLI:\n  $ wsk project deploy --config <config file> --manifest path/to/manifest.yaml\nOr specify all three on CLI:\n  $ wsk project deploy --apihost HOST --auth KEY --namespace NAMESPACE --manifest path/to/manifest.yaml"
```

Comment:
If current directory doesnt have any manifest file->
If the working directory doesn't contain a manifest file

Reply:
where you use "wsk project deploy" in the help message it may be useful to put it in quotes. 
but that's an awfully long message though.

Reply:
Please add `.` before every new line.

Reply:
> where you use "wsk project deploy" in the help message it may be useful to put it in quotes.
> but that's an awfully long message though.

yeah mainly for users to quickly get started with the command instead of referring to any readme/tutorial.